code/API_definitions/qod-api.yaml

```diff
@@ -329,18 +402,238 @@ components:
       description: |
         Ports may be specified as a list of ranges or single ports.
     QosProfile:
+      description: |
+        Data type with attributes of a QosProfile
+      type: object
+      properties:
+        name:
+          allOf:
+            - $ref: "#/components/schemas/QosProfileName"
+            - description: |
+                The name of the QoS profile.  This name is used to identify the profile when creating a session.
+                The name must be unique.
+        description:
+          description: |
+            A description of the QoS profile.
+          type: string
+          example: "QoS profile for video streaming"
+        status:
+          allOf:
+            - $ref: "#/components/schemas/QosProfileStatusEnum"
+            - description: |
```

Comment:
To keep this clarification it has to be moved to the description in `QosProfileStatusEnum`